packages/react-composites/src/composites/MeetingComposite/MeetingComposite.tsx

```diff
@@ -102,20 +104,44 @@ const MeetingScreen = (props: MeetingScreenProps): JSX.Element => {
 
   const [currentMeetingState, setCurrentMeetingState] = useState<CallState>();
   const [currentPage, setCurrentPage] = useState<MeetingCompositePage>();
-  meetingAdapter.onStateChange((newState) => {
-    setCurrentPage(newState.page);
-    setCurrentMeetingState(newState.meeting?.state);
-  });
-
+  const [unreadChatMessages, setUnreadChatMessages] = useState<number>(0);
```

Comment:
the naming of this state implies a collection of messages when its really state that represents the number of unread messages

lets change it to numberOfUnreadChatMessages, setNumberOfUnreadChatMessages or if we can be consistent and we used unreadChatMessagesCount then we should use 

const[ unreadChatMessagesCount, setUnreadChatMessagesCount] = useState<number>(0);